yb-voyager/cmd/cutoverToTargetCmd.go

```diff
@@ -55,6 +56,26 @@ var cutoverToTargetCmd = &cobra.Command{
 				utils.ErrExit("Live migration with Fall-forward workflow is already started on this export-dir. So --prepare-for-fall-back is not applicable.")
 			}
 		}
+		isFallForwardOrFallBackEnabled := bool(prepareForFallBack) || msr.FallForwardEnabled
```

Comment:
@ShivanshGahlot can you pls help review this block?

Reply:
Yeah this looks good to me @priyanshi-yb